modules/react/menu/lib/MenuItem.tsx

```diff
@@ -128,6 +185,63 @@ const StyledItem = styled(Box.as('button'))<StyledType>(
         display: 'flex',
       };
     }
+  },
+  ({isFocused, isDisabled}) => {
+    if (!isFocused && !isDisabled) {
+      return {
+        backgroundColor: 'inherit',
+        '&:hover': {
+          backgroundColor: commonColors.hoverBackground,
+          color: colors.blackPepper300,
+          '.wd-icon-fill, .wd-icon-accent, .wd-icon-accent2': {
+            fill: iconColors.hover,
+          },
+        },
+      };
+    } else if (isDisabled && !isFocused) {
+      return {
+        color: colors.licorice100,
+        cursor: 'default',
+      };
+    } else {
+      // Is focused or focused and disabled
+      return {
+        backgroundColor: isDisabled ? colors.blueberry200 : commonColors.focusBackground,
+        color: typeColors.inverse,
+        'span .wd-icon-background ~ .wd-icon-accent, .wd-icon-background ~ .wd-icon-accent2': {
+          fill: isDisabled ? iconColors.disabled : iconColors.active,
+        },
+        '&:hover': {
+          'span .wd-icon-fill, span .wd-icon-accent, span .wd-icon-accent2': {
+            fill: iconColors.inverse,
+          },
+          'span .wd-icon-background ~ .wd-icon-accent, span .wd-icon-background ~ .wd-icon-accent2': {
+            fill: isDisabled ? iconColors.disabled : iconColors.active,
+          },
+          'span .wd-icon-background': {
+            fill: iconColors.inverse,
+          },
+        },
+        [`[data-whatinput='mouse'] &,
+          [data-whatinput='touch'] &,
+          [data-whatinput='pointer'] &`]: {
+          backgroundColor: 'inherit',
+          color: colors.blackPepper300,
+          'span .wd-icon-background ~ .wd-icon-accent, span .wd-icon-background ~ .wd-icon-accent2': {
+            fill: iconColors.standard,
+          },
+          '&:hover': {
+            backgroundColor: commonColors.hoverBackground,
+            'span .wd-icon-fill, span .wd-icon-accent, span .wd-icon-accent2': {
+              fill: iconColors.hover,
+            },
+          },
+          '.wd-icon-fill, .wd-icon-accent, .wd-icon-accent2': {
+            fill: iconColors.standard,
+          },
+        },
+      };
+    }
```

Comment:
These styled differ from what's in the main package no. It looks like the hover effect is different.

[Example](https://recordit.co/kc9UwZ6I9g) of what is in this PR

Reply:
in Main now https://workday.github.io/canvas-kit/?path=/story/components-popups-menu--basic

Reply:
Let me take a look at updating this